uaclient/cli.py

```diff
@@ -165,36 +178,33 @@ def action_disable(args, cfg):
         return 1
 
 
+@assert_attached_root
```

Comment:
Does this mean we can drop the checks that `assert_attached_root` now performs from `UAEntitlement.can_enable`?

Reply:
I think we should be able to drop those checks from Entitlement.can_enable. no sense duplicating

Reply:
dropped from entitlements. made for a good number of mock drops on getuid there. Thanks for thinking of this.